### ISSUE TITLE
Screenshots with image placement examples do not use the template

### ISSUE THREAD

**Issue by alfredocambera** [funkydan2/hugo-kiera#17]
Reference images are confusing as they do not use hugo-kiera

**Comment by funkydan2** [comment]
I don't understand. The images [in the readme]https://github.com/funkydan2/hugo-kiera#images) look like screenshots of Kiera to me (you can see the raw files [here](https://github.com/funkydan2/hugo-kiera/tree/master/images/screenshots). 

And if you build the example site by running `hugo server --theme="../.."` from the `exampleSite` directory, you can see how these shortcodes work using 'filler' images.

**Comment by alfredocambera** [comment]
They don't look the same to me. Maybe is a thing of my monitor but the headings font looks different https://imgur.com/a/Z02TeOC

Could you please give me your opinion?

**Comment by funkydan2** [comment]
Yep, you're probably right. If you look at the [history of the CSS file](https://github.com/funkydan2/hugo-kiera/commits/master/static/css/styles.css), the base font was changed in November 2017, and screenshots are older than that.

If you're able to redo the screenshots (probably not with the blank images as the example site does...but something nice like what's currently there) and submit a PR, I'll merge.

**Comment by alfredocambera** [comment]
Sure, I'll do it tomorrow.
